api/MediaDevices.json

```diff
@@ -367,11 +367,11 @@
           "support": {
             "chrome": [
               {
-                "version_added": "52"
+                "version_added": "53"
               },
               {
                 "version_added": "47",
-                "version_removed": "52",
+                "version_removed": "53",
                 "flags": [
```

Comment:
@ddbeck a policy question for you before I can finish this change. The presence of the flag data here is inconsistent between api.MediaDevices.getUserMedia and api.Navigator.getUserMedia, and to my eye have made the data worse, as the "navigator.webkitGetUserMedia" note is on the flag entry, where it should be on the main one if it's worth having at all.

In this situation, is it OK to remove the flag data entirely, or should I make everything consistent by adding it to more places? I've also commented on https://github.com/mdn/browser-compat-data/issues/3318.

Reply:
The note feels easy to me: I'd drop it entirely (it seems pointless to refer people _from_ a standard feature to a non-standard, prefixed feature).

As for the flag data, I'm less sure. You said the flag data is inconsistent. The implication here is that when `api.MediaDevices.getUserMedia` was behind a flag, so too was `api.Navigator.getUserMedia`? Is that correct? In general, I'd say make the flags consistent, though I wouldn't obligate you to change to `api.Navigator.getUserMedia` in a PR for `api.MediaDevices.getUserMedia` (which is to say, if you don't want to fuss with `Navigator`, you don't have to).

But!

The details here though point to a narrow case for dropping some flag data: flags for features which are both `"deprecated": true` and `"standard_track": false` are the summit (or is that nadir?) of obscurity. There's really no plausible story for web developers to turn on that preference (there's no standard discussion to contribute to, for example). Maybe that's a partial solution to #3318; I'll open an issue for it.

Reply:
That's right, `api.MediaDevices.getUserMedia` and `api.Navigator.getUserMedia` were behind the same flag in Chromium and were shipped together. I'm not certain they were *added* behind a flag at the same time, and that's the sort of thing that doesn't feel very productive to research.

Unfortunately, the narrow case doesn't apply here, because the flags are on the `api.MediaDevices.getUserMedia` entry. I would argue there is still no plausible reason fro web developers to want to turn on that flag in an old version of Chrome, however.

As for the "Older versions of Chrome" notes, I guess those are to let people know that they might still be able to use this API under a different name. It serves the same purpose as a `prefix` or `alternative_name` entry, but in this case it was on a different interface so that's not possible.

Reply:
Yeah, I discovered that my partial solution involving deprecations isn't all that useful (the first match seemed kind of exceptional, which wasn't very helpful).

I have a comment on the note that remains. I'll start a new line comment for that.